src/saveJSON.js

```diff
@@ -1,5 +1,10 @@
-import { writeFileSync } from 'fs';
+import { writeFile } from 'fs';
 
 export default function saveJSON(cssFile, json) {
-  writeFileSync(`${ cssFile }.json`, JSON.stringify(json));
+  return new Promise((resolve, reject) => {
+    writeFile(
+      `${ cssFile }.json`,
+      JSON.stringify(json),
+      e => e ? reject(e) : resolve());
```

Comment:
It can be useful to resolve the promise with the received json. Could you please make this?